hazelcast-client/src/main/java/com/hazelcast/client/cache/impl/AbstractClientInternalCacheProxy.java

```diff
@@ -331,8 +343,7 @@ protected void onRemoveAsyncInternal(Data keyData, ClientDelegatingFuture future
         addCallback(delegatingFuture, callback);
     }
 
-    protected <T> ICompletableFuture<T> replaceAndGetAsyncInternal(K key, V oldValue, V newValue, ExpiryPolicy
-            expiryPolicy,
+    protected <T> ICompletableFuture<T> replaceAndGetAsyncInternal(K key, V oldValue, V newValue, ExpiryPolicy expiryPolicy,
```

Comment:
This method was asynchronous before, since it always returns a future. The parameter `async` just determined if the statistics should be updated asynchronously. So I changed the method name accordingly.